playerbot/PlayerbotAI.h

```diff
@@ -248,6 +248,14 @@ enum BotRoles
     BOT_ROLE_DPS = 0x04
 };
 
+enum class QuestRewardOptionType : uint8
```

Comment:
Move this to QuestValues.h or questrewardactions.h

Reply:
done & moved to `QuestValues.h`